system/include/platform.h

```diff
@@ -171,6 +171,8 @@ typedef memsize_t rowsize_t;
 #define GetSharedProcedure(h,name) GetProcAddress(h,(char *)name)
 #define LoadSucceeded(h)           ((memsize_t)h >= 32)
 #define GetSharedObjectError()     GetLastError()
+#define GetSharedObjectErrorString() strerror(errno)
+#define CloseSharedObject(h)        FreeLibrary(h)
```

Comment:
This is not a good idea - should be calling FreeSharedObject (in jutil)

Reply:
Good suggestion. Will do